Tensile/BuildCommands/SharedCommands.py

```diff
@@ -0,0 +1,40 @@
+import subprocess
+from pathlib import Path
+from typing import Union
+
+from ..Common import tPrint
+
+
+def compressCodeObject(
+    coPathSrc: Union[Path, str], coPathDest: Union[Path, str], gfx: str, bundler: str
+):
+    """Compresses a code object file using the provided bundler.
+
+    Args:
+        coPathSrc: The source path of the code object file to be compressed.
+        coPathDest: The destination path for the compressed code object file.
+        gfx: The target GPU architecture.
+        bundler: The path to the Clang Offload Bundler executable.
+
+    Raises:
+        RuntimeError: If compressing the code object file fails.
+    """
+    args = [
+        bundler,
+        "--compress",
+        "--type=o",
+        "--bundle-align=4096",
+        f"--targets=host-x86_64-unknown-linux,hipv4-amdgcn-amd-amdhsa--{gfx}",
```

Comment:
Don't you have a code object version format that might be relevant here?

Reply:
I've been in contact with the compiler team and there was no mention of a specific code object version being applied. Further, there is no information on a code object version in the help string for the offload bundler.

Reply:
`hipv4-`? shows up in the triple.